modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/scroll/ElasticSearchScroll.java

```diff
@@ -48,13 +51,17 @@ public boolean hasNext() {
     @Override
     protected boolean fetch() {
         ElasticSearchService ess = Framework.getService(ElasticSearchService.class);
-        if (esScroll == null) {
-            esScroll = ess.scroll(new NxQueryBuilder(session).nxql(request.getQuery())
-                                                             .limit(request.getSize())
-                                                             .onlyElasticsearchResponse(),
-                    request.getTimeout().toSeconds());
-        } else {
-            esScroll = ess.scroll(esScroll);
+        try {
+            if (esScroll == null) {
+                esScroll = ess.scroll(new NxQueryBuilder(session).nxql(request.getQuery())
+                                .limit(request.getSize())
+                                .onlyElasticsearchResponse(),
+                        request.getTimeout().toSeconds());
```

Comment:
There's a format issue:
```suggestion
                esScroll = ess.scroll(new NxQueryBuilder(session).nxql(request.getQuery())
                                                                 .limit(request.getSize())
                                                                 .onlyElasticsearchResponse(),
                        request.getTimeout().toSeconds());
```